include/miopen/miopen.h

```diff
@@ -346,12 +346,11 @@ MIOPEN_DECLARE_OBJECT(miopenReduceTensorDescriptor);
  */
 typedef enum
 {
-    miopenHalf  = 0, /*!< 16-bit floating point (Fully supported) */
-    miopenFloat = 1, /*!< 32-bit floating point (Fully supported) */
-    miopenInt32 = 2, /*!< 32-bit int point (Partially supported) */
-    miopenInt8  = 3, /*!< 8-bit int point (Partially supported) */
-    miopenInt8x4 =
-        4, /*!< Pack of four 8-bit int points in NCHW_VECT_C format (Partially supported) */
+    miopenHalf     = 0, /*!< 16-bit floating point (Fully supported) */
+    miopenFloat    = 1, /*!< 32-bit floating point (Fully supported) */
+    miopenInt32    = 2, /*!< 32-bit int point (Partially supported) */
+    miopenInt8     = 3, /*!< 8-bit int point (Partially supported) */
```

Comment:
```suggestion
    miopenInt32    = 2, /*!< 32-bit integer (Partially supported) */
    miopenInt8     = 3, /*!< 8-bit integer (Partially supported) */
```

Reply:
Nice catch. Originated from https://github.com/atamazov/MIOpen/commit/4f6f1cb16a7b7ad18d0e3d2c225109d7da3b2663#diff-509a445a5b37e34b16d382a095f1828a375119107fdf97f5b5f95ce93857e305

Reply:
Resolved in [e6b92e7](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2452/commits/e6b92e75eef156410d573fa7c386b79828aa4de0)